littlelines/otwarchive
# Ao3 5023 rails 4 dot 2 bugfix

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-5023

## Purpose

What does this PR do?

This PR fixes the issue where the same collection was showing up multiple times on a given work's collections page.

## Testing

How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)

In staging - go to http://test.archiveofourown.org/works/1064145/collections and ensure that the collection ModeratedUnrevealedTwoWorks shows up only once.

Locally - 

1. Find a collection in the console that has more than one work
2. Go to the collections page for one of those works (/works/work_id/collections)
3. Check that the collection found in step 1 only shows up once

## References

Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](http://archiveofourown.org/admin_posts?tag=1)?*

Wendy Randquist - Littlelines

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

she/her

(if you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender)
